### PR TITLE
Address comments from @reidmix's reviews

### PR DESCRIPTION
**Pull request by lexelby**
This addresses some various comments/suggestions from @reidmix.  It was faster to just create a PR than to try to make cards.  I'll merge this when I'm no longer on interrupts.

### CODE REVIEW

File: lib/chat_ops/chat_ops.rb [hunk 1]
```diff
@@ -178,7 +179,7 @@ def find_and_remove_names!(message)
     end
 
     def handles_regex
-      handles = User.pluck(:handle).reject(&:nil?).reject(&:empty?)
+      handles = User.with_handle.pluck(:handle)
```
Comment (joshuatobin) [inline lib/chat_ops/chat_ops.rb]: Thats really nice <#

File: lib/chat_ops/command.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module ChatOps
-  class ChatOpsCommand
+  class Command
```
Comment (joshuatobin) [inline lib/chat_ops/command.rb]: + this reads a bit easier.

**Comment by joshuatobin** [comment]
@lexelby nice job. 👍

**Comment by reidmix** [comment]
Yes, so many 👍

**Pull request merged**